modules/context/access_log.go

```diff
@@ -67,17 +68,22 @@ func AccessLogger() func(http.Handler) http.Handler {
 				requestID = parseRequestIDFromRequestHeader(req)
 			}
 
+			reqAddr, _, err := net.SplitHostPort(req.RemoteAddr)
+			if err != nil {
+				reqAddr = req.RemoteAddr
+			}
+
 			next.ServeHTTP(w, r)
 			rw := w.(ResponseWriter)
 
 			buf := bytes.NewBuffer([]byte{})
-			err := logTemplate.Execute(buf, routerLoggerOptions{
+			err = logTemplate.Execute(buf, routerLoggerOptions{
 				req:            req,
 				Identity:       &identity,
 				Start:          &start,
 				ResponseWriter: rw,
 				Ctx: map[string]interface{}{
-					"RemoteAddr": req.RemoteAddr,
+					"RemoteAddr": reqAddr,
```

Comment:
Instead of changing the meaning, is it better to add a new key: RemoteHost ?

IMO Golang's RemoteAddr concept is the host with port.

Reply:
Thanks for the feedback wxiaoguang :+1: